### DISCUSSION TITLE
Error installing packages included in scripts

### DISCUSSION THREAD

**Discussion by TGriffiths88** [johnlindquist/kit#1081] [Errors] [answered]
I'm trying to use 'playwright' in a script and when I launch the script from the prompt, it correctly prompts to install the missing dependency. But when I go ahead and select 'Install Playright' it fails with the following error:

```
$ cd /Users/tomgriffiths/.npm/_npx/3415c6bfffe36ec5/node_modules/@johnlindquist/kit
[09:49:27.992] [info] 
    at process.processTicksAndRejections (node:internal/process/task_queues:96:5)
    at file:///Users/tomgriffiths/.kit/node_modules/@johnlindquist/globals/dist/index.js:28:123014
    at makeError (file:///Users/tomgriffiths/.kit/node_modules/@johnlindquist/globals/dist/index.js:28:111641)
npm ERR!     /Users/tomgriffiths/.npm/_logs/2023-01-31T09_49_27_555Z-debug-0.log
npm ERR! A complete log of this run can be found in:

npm ERR! ENOTDIR: not a directory, rename '/Users/tomgriffiths/.kenv/node_modules/@johnlindquist/kit' -> '/Users/tomgriffiths/.kenv/node_modules/@johnlindquist/.kit-2BZClemj'
npm ERR! errno -20
npm ERR! dest /Users/tomgriffiths/.kenv/node_modules/@johnlindquist/.kit-2BZClemj
npm ERR! path /Users/tomgriffiths/.kenv/node_modules/@johnlindquist/kit
npm ERR! syscall rename
npm ERR! code ENOTDIR
[09:49:27.989] [warn] Error: Command failed with exit code 236: npm i -D playwright
[09:49:27.988] [warn] ☠️ ERROR PROMPT SHOULD SHOW ☠️
```

Any ideas where to look to solve this issue?

EDIT: Just did a fresh re-install and its resolved the issue. Something must have gone awry in the original installation process

**Comment by johnlindquist** [comment, marked as answer]
> EDIT: Just did a fresh re-install and its resolved the issue. Something must have gone awry in the original installation process

Glad you got it sorted. Let me know if you can think of anything that may have caused it the first time though.

**Comment by tegomass** [comment]
Same here, reinstalling kit solve the issue. (from windows x64 1.40.62 to 1.48.0)
